src/lib/cmp.js

```diff
@@ -224,16 +226,16 @@ export default class Cmp {
 		// Special case where we have the full CMP implementation loaded but
 		// we still queue these commands until there is data available. This
 		// behavior should be removed in future versions of the CMP spec
-		else if (
-			(!this.store.persistedVendorConsentData && (command === 'getVendorConsents' || command === 'getConsentData')) ||
-			(!this.store.persistedPublisherConsentData && command === 'getPublisherConsents')) {
-			log.info(`Queuing command: ${command} until consent data is available`);
-			this.commandQueue.push({
-				command,
-				parameter,
-				callback
-			});
-		}
+		// else if (
```

Comment:
Is there a reason we commented this out instead of erasing it? Usually commenting out chunks is a bit of a code smell

Reply:
I left it commented out because the upstream CMP has a "todo" to remove this, since i've already removed it i want the comment here so it matches up with the upstream file and context is given